src/content/developers/docs/programming-languages/dart/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ethereum for Dart Developers
+description: Learn how to develop for Ethereum using the Dart language
+lang: en
+sidebar: true
+incomplete: true
+---
+
+## Getting Started with Smart Contracts and the Solidity Language {#getting-started-with-smart-contracts-and-solidity}
+
+## Tutorials {#tutorials}
+
+1. [This tutorial](https://www.geeksforgeeks.org/flutter-and-blockchain-hello-world-dapp/) takes you through all the steps to get started:
+   1. Installing the [Truffle development suite](https://www.trufflesuite.com/)
+   2. Writing a smart contract in [Solidity](https://soliditylang.org/)
+   3. Writing a user interface in Dart
+2. [This tutorial](https://medium.com/dash-community/building-a-mobile-dapp-with-flutter-be945c80315a) is a lot shorter, which might be better
+   if you already know the basics.
+4. If you prefer the learn by watching a video, [watch this tutorial](https://www.youtube.com/watch?v=3Eeh3pJ6PeA), which is about an hour long
+5. If you are impatient, you might prefer [this tutorial](https://www.youtube.com/watch?v=jaMFEOCq_1s), which is only about twenty minutes
```

Comment:
Maybe pull some of the tutorial names into the links to avoid all the links saying "this tutorial."